src/dlgTriggerEditor.cpp

```diff
@@ -1378,7 +1378,7 @@ void dlgTriggerEditor::slot_searchMudletItems(const int index)
     mpSourceEditorEdbee->controller()->update();
 }
 
-void dlgTriggerEditor::searchVariables(const QString& s)
+void dlgTriggerEditor::searchVariables(const QString& text)
```

Comment:
I'd suggest `needle` for this term in this and the following methods on the old "searching for a needle in a haystack metaphor"...! :grin:

Reply:
It would work, but `text` is more intuitive to keep in your head.

Reply:
We already use that metaphor though, consider the following, one of several in the `TTrigger` class:
```cpp
bool TTrigger::match_begin_of_line_substring(const QString& haystack, const QString& needle, int patternNumber, int posOffset)
{
    if (haystack.startsWith(needle)) {
        processBeginOfLine(needle, patternNumber, posOffset);
        return true;
    }
    return false;
}
```

Reply:
There is no haystack here, though... I think `text` will be fine for most folks